server/src/map/map.controller.js

```diff
@@ -0,0 +1,49 @@
+const httpStatus = require('http-status');
+const map = require('./map.helper');
+
+function get(req, res) {
+  map.getAllMaps()
+    .then(maps => {
+      return res.send(maps);
```

Comment:
the idea to use args via closure looks nice for me, but it's realization is not so fluence:)
My option can be next:

```
let connectionDBtoServerHelper = (dbSelection, res) => dbSelection
    .then(res.send)
    .catch(res.status(ERROR_CODE).send);

// and how to use:
let get = (req, res) => connectionDBtoServerHelper(map.getAllMaps(), res);

```